holoviews/plotting/bokeh/element.py

```diff
@@ -104,6 +103,12 @@ class ElementPlot(BokehPlot, GenericElementPlot):
           * timeout   - Timeout (in ms) for checking whether interactive
                         tool events are still occurring.""")
 
+    sizing_mode = param.ObjectSelector(default='fixed',
+                                       objects=[
+            'fixed', 'scale_width', 'scale_height',
+            'scale_both', 'stretch_both'
+        ], doc="""Defines how the plot scales when resized.""")
+
```

Comment:
Agreed, I'll add comments, but the usage of WIP across all the projects I've encountered has meant not ready for review, and ``ready`` means ready for review not necessarily merge.